app/models/variant_override.rb

```diff
@@ -21,6 +19,12 @@ class VariantOverride < ActiveRecord::Base
     where(hub_id: hubs)
   }
 
+  scope :distinct_import_dates, lambda {
+    select('DISTINCT variant_overrides.import_date').
+      where('variant_overrides.import_date IS NOT NULL').
+      order('import_date DESC')
```

Comment:
just as a minor detail, could we rely on ActiveRecord's hash style? So it'd be like `order(import_date: :desc)`. It decouples the scope from SQL a bit.

See the examples in https://apidock.com/rails/ActiveRecord/QueryMethods/order.

Reply:
Apparently this also works only after Rails 4 :/ as well as `.where.not()` which could have been used in the line above

Reply:
Really? :sob: